### PR TITLE
Control DANDI Jobs and Threads

### PR DESCRIPTION
**Pull request by garrettmflynn**
This PR allows users to control the number of jobs and threads when uploading to DANDI on the GUIDE.

May fix the segmentation fault issue we discovered during alpha tests.

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn Is there a way to constrain the values of the IntRangeSlider to not include zero? Also to include in the description something about how `-1` syntax uses all available processes, etc...

**Comment by garrettmflynn** [comment]
Not yet—but I will make a way. Users can also free-input any flat value—so I'll implement a way to automatically round those too.

**Comment by garrettmflynn** [comment]
Working on this now. Is there a similar way to bound threads?

**Comment by garrettmflynn** [comment]
Just pushed the proper workflow for jobs. Threads currently have no maximum

**Comment by CodyCBakerPhD** [comment]
> Is there a similar way to bound threads?

Yessir,

```python
number_of_threads = cpu_count() / cpu_count(logical=False)
```

**Comment by garrettmflynn** [comment]
Sweet updated. Should be ready for review now

File: schemas/json/dandi/upload.json [hunk 1]
```diff
@@ -4,6 +4,20 @@
...
+            "type": "integer",
+            "title": "Job Count",
+            "description": "The number of files to upload in parallel. A value of <code>-1</code> uses all available processes",
+            "default": 1,
+            "min": -1
+        },
+        "number_of_threads": {
+            "type": "integer",
+            "title": "Threads per Job",
+            "description": "The number of threads to handle each file.",
```
Comment (CodyCBakerPhD) [inline schemas/json/dandi/upload.json:17]:
```suggestion
            "description": "The number of threads to handle each file. A value of <code>-1</code> uses all available threads per process.",
```

**Comment by CodyCBakerPhD** [comment]
One minor update to a description and otherwise LGTM

**Comment by garrettmflynn** [comment]
Awesome. Glad we're getting this in

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn One question on the effect of the new chromatic stories

**Comment by garrettmflynn** [comment]
Working locally now on Storybook

**Pull request merged**